diff -ru a/Panel.qml b/Panel.qml
--- a/Panel.qml	2026-10-02 11:03:30.703602237 -0500
+++ b/Panel.qml	2026-10-02 11:03:30.782359014 -0500
@@ -470,6 +470,7 @@
     anchors.fill: parent
     bar: root.bar
     text: Quickshell.screens.length > 1 ? "󰍺" : "󰍹"
+    iconComponent: BarSvgIcon { host: button }
     onPressed: function(b) { root.toggle() }
     onWheelMoved: function(delta) {
       if (!root.brightnessAvailable) return
