diff -ru a/BarWidget.qml b/BarWidget.qml
--- a/BarWidget.qml	2026-10-02 11:34:34.622095542 -0500
+++ b/BarWidget.qml	2026-10-02 11:34:42.600928707 -0500
@@ -70,6 +70,7 @@
     bar: root.bar
     text: panelLoader.item ? panelLoader.item.label : ""
     slotSize: Style.bar.statusSlot
+    iconComponent: BarSvgIcon { host: button }
     // Tooltip suppressed because the panel is the detail view.
     tooltipText: ""
 
