diff -ru a/BarWidget.qml b/BarWidget.qml
--- a/BarWidget.qml	2026-10-01 22:10:18.000000000 -0500
+++ b/BarWidget.qml	2026-10-02 11:57:31.802427607 -0500
@@ -450,10 +450,8 @@
       spacing: Style.space(6)
 
       // Phase Icon
-      Text {
-        text: root.currentIcon
-        font.family: Style.font.family
-        font.pixelSize: Style.font.body
+      BarSvgIcon {
+        hostText: root.currentIcon
         color: root.isSessionActive ? root.currentPhaseColor : root.barForeground
         opacity: (root.phase === "idle" || root.paused) ? 0.75 : 1.0
         anchors.verticalCenter: parent.verticalCenter
@@ -479,10 +477,8 @@
       anchors.centerIn: parent
       spacing: Style.space(2)
 
-      Text {
-        text: root.currentIcon
-        font.family: Style.font.family
-        font.pixelSize: Style.font.body
+      BarSvgIcon {
+        hostText: root.currentIcon
         color: root.isSessionActive ? root.currentPhaseColor : root.barForeground
         anchors.horizontalCenter: parent.horizontalCenter
       }
